src/mongo/db/range_deleter.cpp

```diff
@@ -129,7 +129,7 @@ static void logCursorsWaiting(RangeDeleteEntry* entry) {
 }
 
 struct RangeDeleter::NSMinMax {
-    NSMinMax(std::string ns, const BSONObj min, const BSONObj max) : ns(ns), min(min), max(max) {}
+    NSMinMax(const std::string& ns, const BSONObj min, const BSONObj max) : ns(ns), min(min), max(max) {}
```

Comment:
A better change here might be to leave the parameter as by-value, and just re-write the member initializer as ```ns(std::move(ns))```